src/frontend/src/common/redux/index.tsx

```diff
@@ -64,6 +66,16 @@ const reduxReducer = (state = getInitialState(), action: AnyAction) => {
           pathogen: payload,
         },
       };
+    case CZGEReduxActions.ADD_NOTIFICATION_ACTION_TYPE:
```

Comment:
I keep thinking about the `reduxId` aspect, and since every payload has to run its own `Date.now()` call, I think it would be better to remove the id from the payload entirely and just have it be generated here in the reducer. Less boilerplate where the notifications gets dispatched and seems like a bit more separation of concerns.

_Devil's Advocate_: That said, if we ever have a non-notif component that needs to kick off a `DELETE_NOTIFICATION_ACTION_TYPE` to delete, having the id get made in the reducer prevents it from being exposed. If we have a reduxId that gets set in some component (say in the tree creation modal), the modal could theoretically have react state to track the reduxId, then fire off a delete based on specific logic. But I don't think we have any use cases like that right now? Also we could have an optional uuid field and the delete action could be expanded to target reduxId OR uuid matches. ... oh! Hmmm, maybe the best of both worlds would be to just have a default `reduxId: Date.now()` in the payload portion for `addNotification` function? Expected default there, but allows for special case overriding.

Reply:
Redux reducers are supposed to be 100% pure and always return the same state given the same input. As such, I do not want to add it to the reducer. But I think it would be ok to add it to the action with the other default props. That would let it still be overwritable.